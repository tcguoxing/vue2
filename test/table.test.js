import testTable from '../src/components/fileForTest/table.vue'
import { mount, shallowMount } from '@vue/test-utils'
import { describe, it, beforeEach, afterEach } from '@jest/globals'

// 测试table表
describe('table测试', () => {
    let wrapper
    it('快照样式', () => {
        wrapper = shallowMount(testTable, {
            propsData: {
                columns: ['tian', 'gan', 'di', 'zhi'],
                tableData: [
                {tian: 'this is tian', gan: 'this is gan', di: 'this is di', zhi:'this is zhi'},
                {tian: 'this is tian', gan: 'this is gan', di: 'this is di', zhi:'this is zhi'},
                {tian: 'this is tian', gan: 'this is gan', di: 'this is di', zhi:'this is zhi'}
            ]
            }
        })
        expect(wrapper.html()).toMatchSnapshot()
    })
})
