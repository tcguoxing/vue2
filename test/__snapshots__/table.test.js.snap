// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table测试 快照样式 1`] = `
"<el-table data=\\"[object Object],[object Object],[object Object]\\" stripe=\\"\\" border=\\"\\">
  <el-table-column prop=\\"tian\\"></el-table-column>
  <el-table-column prop=\\"gan\\"></el-table-column>
  <el-table-column prop=\\"di\\"></el-table-column>
  <el-table-column prop=\\"zhi\\"></el-table-column>
</el-table>"
`;
